packages/react/src/components/Renderer.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import type { NovuUI, NovuUIOptions } from '@novu/js/ui';
+import ReactDOM from 'react-dom';
+import { MountedElement, RendererProvider } from '../context/RenderContext';
+
+type RendererProps = React.PropsWithChildren<{
+  options: NovuUIOptions;
+}>;
+
+export const Renderer = (props: RendererProps) => {
+  const { options, children } = props;
+  const [novuUI, setNovuUI] = React.useState<NovuUI | undefined>();
+  const [mountedElements, setMountedElements] = React.useState(new Map<HTMLElement, MountedElement>());
+
+  const mountElement = React.useCallback(
+    (el: HTMLElement, mountedElement: MountedElement) => {
+      setMountedElements((prev) => {
+        const newMountedElements = new Map(prev);
+        newMountedElements.set(el, mountedElement);
+
+        return newMountedElements;
+      });
+
+      return () => {
+        setMountedElements((prev) => {
+          const newMountedElements = new Map(prev);
+          newMountedElements.delete(el);
+
+          return newMountedElements;
+        });
+      };
+    },
+    [setMountedElements]
+  );
+
+  React.useEffect(() => {
+    const loadNovuUI = async () => {
+      const { NovuUI } = await import('@novu/js/ui');
+      const ui = new (NovuUI as new (novuOptions: NovuUIOptions) => NovuUI)(options);
```

Comment:
```suggestion
      const { NovuUI } = await import('@novu/js/ui');
```

Reply:
Sorry, i didn't understood what I need to change here